functions/helper_lib.sh

```diff
@@ -112,12 +112,22 @@ get_docker_configuration_file() {
   CONFIG_FILE='/dev/null'
 }
 
+if command -v jq &> /dev/null; then
+  HAVE_JQ=true
+else
+  HAVE_JQ=false
+fi
+
 get_docker_configuration_file_args() {
   OPTION="$1"
 
   get_docker_configuration_file
 
-  grep "$OPTION" "$CONFIG_FILE" | sed 's/.*://g' | tr -d '" ',
+  if "$HAVE_JQ"; then
+    jq --monochrome-output --raw-output ".[\"${OPTION}\"]" "$CONFIG_FILE"
+  else
+    cat "$CONFIG_FILE" | tr -u { '\n' | tr , '\n' | tr } '\n' | grep "$OPTION" | sed 's/.*://g' | tr -d '" ',
```

Comment:
```sh
cat /tmp/daemon.json | tr -u { '\n' | tr , '\n' | tr } '\n' | grep "userland-proxy" | sed 's/.*://g' | tr -d '" ',
tr: invalid option -- 'u'
Try 'tr --help' for more information.
$ tr --version
tr (GNU coreutils) 8.32
Copyright (C) 2020 Free Software Foundation, Inc.
```

Reply:
Good catch. Looks like the busybox `tr` doesn't have that option either. I think it can be safely removed, it is just to guarantee unbuffered output which should not matter here.

Reply:
yeah, without `-u` it works great.

Reply:
Changed in 79407ce